js/interface.js

```diff
@@ -901,10 +922,16 @@ function saveCurrentData() {
 
   var payload = getCommitPayload(entries);
 
+  const HistoryStack = Fliplet.Registry.get('history-stack');
+  const { deleteColumns, renameColumns } = HistoryStack.columnsInfo.getCommitPayload();
```

Comment:
I prefer making this part of the `getCommitPayload` function above so that it returns the `deleteColumns` and `renameColumns` properties as well